src/components/Reports/ReportView.tsx

```diff
@@ -12,9 +12,13 @@ import BackToLink from 'src/components/common/BackToLink';
 import ReportFormAnnual from 'src/components/Reports/ReportFormAnnual';
 import useSnackbar from 'src/utils/useSnackbar';
 import ConcurrentEditorWarningDialog from 'src/components/Reports/ConcurrentEditorWarningDialog';
+import useQuery from 'src/utils/useQuery';
 
 export default function ReportView(): JSX.Element {
   const { reportId } = useParams<{ reportId: string }>();
+
+  const query = useQuery();
+  const invalidEditor = !!query.get('invalidEditor');
```

Comment:
we should remove the param.. so reloading the page won't keep showing that message